README.md

```diff
@@ -24,9 +24,10 @@ See [Testing Farm docs](https://docs.testing-farm.io) for more information on su
 
 ### Testing Farm
 
-| Input Name                   | Description                                                                                                                       | Default value                 |
-|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
-| `api_key`                    | Testing farm API key.                                                                                                             | empty, **required from user** |
+| Input Name | Description | Default value |
+|------------|-------------|---------------|
+| `api_key`  | Testing farm API key | empty, **required from user** |
```

Comment:
What about to add there a note about onboarding document? https://docs.testing-farm.io/general/0.1/onboarding.html
So if users want to use Testing Farm, then they can easily ask for access.

Reply:
Good point. I can create separate PR for that.